manifests/plugin/python.pp

```diff
@@ -47,6 +50,13 @@
     globals  => $globals,
   }
 
+  if $manage_python {
+    ensure_packages([$python_package], { ensure => $python_package_ensure })
```

Comment:
I don't like this because it's really a soft dependency and is not some kind of separate `collectd_python` package. For example, if a plugin required mongodb, apache or ruby would it be appropriate to manage installation in this module? That being said, `ensure_packages` is the best way to do it.

Reply:
It's slightly different because it's the fact and puppet manifest that requires 'python'. collectd itself does not require it.
Anyway especially with the default false the code is basically pointless so lets just chop it.